roles/confluent.variables/defaults/main.yml

```diff
@@ -20,6 +20,166 @@ kafka_rest_jolokia_port: 7775
+broker_cleaner_appender_max_log_files: 10
+broker_cleaner_appender_log_file_size: 100MB
+
+broker_controller_appender_log_path: /var/log/kafka/
+broker_controller_appender_log_name: controller.log
+broker_controller_appender_max_log_files: 10
+broker_controller_appender_log_file_size: 100MB
+
+broker_authorizer_appender_log_path: /var/log/kafka/
+broker_authorizer_appender_log_name: kafka-authorizer.log
+broker_authorizer_appender_max_log_files: 10
+broker_authorizer_appender_log_file_size: 100MB
+
+## Garbage Collection logging
+
+broker_gclog_path: /var/log/kafka/
+broker_gclog_name: broker-gc.log
+broker_gclog_max_log_files: 10
+broker_gclog_log_file_size: 100M
+
+## Schema Registry logging configuration
+
+## log4j configuration
+
+schema_registry_appender_log_path: /var/log/confluent/schema-registry/
+schema_registry_appender_log_name: schema-registry.log
+schema_registry_appender_max_log_files: 10
+scheam_registry_appender_log_file_size: 100MB
+
+## Garbage Collection logging
+
+schema_registry_gclog_path: /var/log/confluent/schema-registry/
+schema_registry_gclog_name: schema-registry-gc.log
+schema_registry_gclog_max_log_files: 10
+schema_registry_gclog_log_file_size: 100M
+
+
+## Connect logging configuration
+
+## log4j configuration
+
+connect_distributed_appender_log_path: /var/log/kafka/
+connect_distributed_appender_log_name: connect-distributed.log
+connect_distributed_appender_max_log_files: 10
+connect_distributed_appender_log_file_size: 100MB
+
+## Garbage Collection logging
+
+connect_distributed_gclog_path: /var/log/kafka/
+connect_distributed_gclog_name: connect-distributed-gc.log
+connect_distributed_gclog_max_log_files: 10
+connect_distributed_gclog_log_file_size: 100M
+
+## KSQL logging configuration
+
+## log4j configuration
+
+ksql_appender_log_path: /var/log/confluent/ksql/
+ksql_appender_log_name: ksql.log
+ksql_appender_max_log_files: 10
+ksql_appender_log_file_size: 100MB
+
+ksql_streams_appender_log_path: /var/log/confluent/ksql/
+ksql_streams_appender_log_name: ksql-streams.log
+ksql_streams_appender_max_log_files: 10
+ksql_streams_appender_log_file_size: 100MB
+
+ksql_kafka_appender_log_path: /var/log/confluent/ksql/
+ksql_kafka_appender_log_name: ksql-kafka.log
+ksql_kafka_appender_max_log_files: 10
+ksql_kafka_appender_log_file_size: 100MB
+
+## Garbage Collection logging
+
+ksql_gclog_path: /var/log/confluent/ksql/
+ksql_gclog_name: ksql-gc-test.log
+ksql_gclog_max_log_files: 10
+ksql_gclog_log_file_size: 100M
+
+## Rest proxy logging configuration
+
+## log4j configuration
+
+rest_proxy_appender_log_path: /var/log/kafka-rest/
+rest_proxy_appender_log_name: rest-proxy.log
+rest_proxy_appender_max_log_files: 10
+rest_proxy_appender_log_file_size: 100MB
+
+## Garbage Collection logging
+
+rest_proxy_gclog_path: /var/log/kafka-rest/
+rest_proxy_gclog_name: rest-proxy-gc.log
+rest_proxy_gclog_max_log_files: 10
+rest_proxy_gclog_log_file_size: 100M
+
+## Control Center logging configuration 
```

Comment:
move component vars into component/defaults/main.yml if they do not need to be global